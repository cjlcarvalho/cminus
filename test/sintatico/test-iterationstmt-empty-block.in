void main(void){
	while(a==1){}
	return;
}
