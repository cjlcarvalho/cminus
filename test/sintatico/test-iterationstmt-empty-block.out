[program 
  [decfunc 
    [void]
    [main]
    [paramlist]
    [block 
      [while 
        [== [var a] [1]]
        [block 
        ]
      ]
      [return ]
    ]
  ]
]
